### PR TITLE
EPMEDU-682: Create Tab Bar for navigation

### PR DESCRIPTION
**Pull request by FlinForm**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -24,12 +24,14 @@ buildscript {
         set("desugarJdkLibs", "1.1.5")
         set("composeVersion", "1.1.1")
         set("activityComposeVersion", "1.4.0")
+        set("navigationComposeVersion", "2.5.0-alpha02")
```
Comment (egorikftp) [inline build.gradle.kts]: Please use stable now

File: app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt [hunk 2]
```diff
@@ -19,12 +34,75 @@ class TabActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
         setContent {
             AnimealTheme {
-                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background) {
-                    Box(modifier = Modifier.fillMaxSize()) {
-                        Text(text = "Hello Animeal!", modifier = Modifier.align(Alignment.Center))
-                    }
+                val navigationController = rememberNavController()
+                val navigationViewModel: NavigationViewModel by viewModels()
```
Comment (egorikftp) [inline app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt]: please extract into field

**Comment by egorikftp** [comment]
Also, please check detekt

File: build.gradle.kts (hunk 1 above)
Comment (FlinForm) [reply build.gradle.kts]: done

File: app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt (hunk 2 above)
Comment (FlinForm) [reply app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt]: done

**Review by egorikftp — APPROVED** [review]

File: app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt [hunk 3]
```diff
@@ -3,28 +3,108 @@ package com.epmedu.animeal.tabs
...
             AnimealTheme {
-                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background) {
-                    Box(modifier = Modifier.fillMaxSize()) {
-                        Text(text = "Hello Animeal!", modifier = Modifier.align(Alignment.Center))
-                    }
+                val navigationController = rememberNavController()
+                Scaffold(
+                    bottomBar = { BottomNavigationBar(navigationController) },
+                    modifier = Modifier.fillMaxSize()
+                ) {
```
Comment (DoTheMonkeyBusiness) [inline app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt]: Please, add padding to scaffold content.

**Pull request merged**